services/version-api/helpers/getVersionConfigurations.js

```diff
@@ -0,0 +1,19 @@
+import { throwError } from '@helsingborg-stad/npm-api-error-handling';
+import to from 'await-to-js';
+
+import params from '../../../libs/params';
+import config from '../../../config';
+
+/**
+ * @returns {Promise<{versions: {ios: {min: string, max: string}, android: {min: string, max: string}}}>} SSM parameters
+ */
+async function getVersionConfigurations() {
+  const [error, response] = await to(params.read(config.version.envsKeyName));
+  if (error) {
+    throwError(500);
```

Comment:
`throwError` only throws implicitely. Would it be ok to add a `return` statement afterwards?

```javascript
throwError(500);
return null;
```

Otherwise we can never be 100% sure.